application-forum-ui/src/main/resources/ForumCode/AnswerSheet.xml

```diff
@@ -53,6 +53,26 @@
         &lt;/dl&gt; 
       &lt;/div&gt;
     {{/html}}
+  #elseif ($request.commentEditForm == 'true')
+    {{html clean="false"}}
+      ## XWIKI-17318
+      #set ($commentEditForm = "#template('commentsinline.vm')")
+      #if (!$commentEditForm.contains('&lt;/textarea&gt;'))
+        ## XWiki 12.6+
+        #set ($discard = $doc.use('XWiki.XWikiComments', $request.number))
+        #set ($commentEditField = $doc.display('comment', 'edit'))
+        #set ($insertPosition = $commentEditForm.indexOf('&lt;!-- loaded by javascript'))
+        #if ($insertPosition != -1)
+          #set ($commentEditBuilder = $commentEditForm.substring(0, $insertPosition) + $commentEditField)
+          #set ($commentEditForm = $commentEditBuilder + $commentEditForm.substring($insertPosition))
```

Comment:
This additional ``commentEditBuilder`` variable is a bit confusing. I think it would be more clean if the insert was done in a single statement.